src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int cargoCapacity;
+    private int numberOfAxles;
+
+    public Truck(int cargoCapacity, int numberOfAxles) {
+        this.cargoCapacity = cargoCapacity;
+        this.numberOfAxles = numberOfAxles;
+    }
+
     public Truck() {
     }
 
+    public int getCargoCapacity() {
```

Comment:
It's good practice to provide a brief comment describing what the method does, even if it seems self-explanatory. This can improve code readability and maintainability.